### PR TITLE
Fix the comments history shows properly in identification(project), third-party, project info tabs

### PR DESCRIPTION
**Pull request by su-ram**
### Fix the bug #106 ###
- change the parameter of function that loads the comments 
- The reason is the wrong path parameter. The function(openCommentHistory) accepts the `url` variable but the former one puts string which is not an url path ("prj"). So I changed that string into correct api path url. 

### Results ###

- before 
It shows none of the comments in identification tab. 
bug106before

- after
It shows correctly. 
bug106after

## Type of change
Please insert 'x' one of the type of change.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/admin/project/common-js.jsp [hunk 1]
```diff
@@ -24,7 +24,7 @@ var com_evt = {
 		
 		$('.btnCommentHistory').on('click', function(e){
 			e.preventDefault();
-			openCommentHistory("prj", "${project.prjId}");
+            openCommentHistory('<c:url value="/comment/popup/prj/${project.prjId}"/>');
```
Comment (riyenas0925) [inline src/main/webapp/WEB-INF/views/admin/project/common-js.jsp:27]: Dear @su-ram 
Indentation is required (change 12 space to 3 tab)

**Comment by riyenas0925** [comment]
Dear @su-ram 

LGTM 👍 

but, it has not been applied to the ```3rd Party List``` yet. could you please commit so that the ```3rd Party List``` can also see the comment history?

image

**Comment by soimkim** [comment]
Dear @su-ram , 
Please change the title of the PR to something about your commit. (instead of the issue number)
Because the PR title will be added to the [CHANGELOG](https://github.com/fosslight/fosslight/blob/develop/CHANGELOG.md) later. 😄

**Review by soimkim — CHANGES_REQUESTED** [review]
Dear @su-ram ,
Please check the indentation.

**Comment by su-ram** [comment]
@soimkim 

I updated 
- [ ] Apply indentations. 
- [ ] Descript the changes in title.
😆

File: src/main/webapp/WEB-INF/views/admin/project/common-js.jsp (hunk 1 above)
Comment (su-ram) [reply src/main/webapp/WEB-INF/views/admin/project/common-js.jsp]: Thanks for the find tips. I applied it. 👍

**Review by soimkim — APPROVED** [review]

**Pull request merged**